src/path_specific.jl

```diff
@@ -0,0 +1,32 @@
+using Omega, Random
+import Omega:intervene, Interventions
+
+export apply_ps_intervention, BlockedEdges
```

Comment:
`BlockedEdges` is this defined?

Reply:
No, not yet. I meant to make it a struct, I'll do it now.